feat: Increase the client redirect_uri length limit

hsinn0 opened the pull request:
- Change the database data type from varchar to text type as mysql varchar type is subject to the table's column size limit

[#183903458]

strehle commented on uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/ClientAdminEndpointsIntegrationTests.java:
```diff
@@ -50,13 +50,7 @@
 import org.springframework.util.MultiValueMap;
 import org.springframework.web.client.RestTemplate;
 
-import java.util.ArrayList;
```
Can you please revert import changes.
There is an ongoning discussion about clean code but we have different view on that but therefore please exclude such changes where they are not needed

hsinn0 replied:
Done

strehle commented on uaa/src/test/java/org/cloudfoundry/identity/uaa/mock/clients/ClientAdminEndpointsMockMvcTests.java:
```diff
@@ -64,15 +64,7 @@
 import org.springframework.web.context.WebApplicationContext;
 
 import javax.servlet.http.HttpServletResponse;
-import java.util.ArrayList;
```
See other comments about import changes

hsinn0 replied:
Done

strehle reviewed (changes requested):
please revert import changes

hsinn0 committed:
feat: Increase the client redirect_uri length limit

- Change the database data type from varchar to text type
as mysql varchar type is subject to the table's column size limit

[#183903458]

Co-authored-by: Jhonathan Aristizabal <jhonathana@vmware.com>
Co-authored-by: Peter Chen <peterch@vmware.com>

peterhaochen47 commented:
Merged as all comments have been addressed.
